<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-CL5VKH3" failures="0" tests="1" name="Default test" time="4.656" errors="0" timestamp="21 May 2021 14:29:10 GMT">
  <testcase classname="jl.project.androidtests.UserRequirement1_Test" name="@BeforeClass setup" time="4.612">
    <failure type="org.openqa.selenium.SessionNotCreatedException" message="Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: The Android SDK root folder &amp;apos;C:\Users\jeanl\AppData\Local\Android\Sdkxxx&amp;apos; does not exist on the local file system. Read https://developer.android.com/studio/command-line/variables for more details
Build info: version: &amp;apos;3.13.0&amp;apos;, revision: &amp;apos;2f0d292&amp;apos;, time: &amp;apos;2018-06-25T15:24:21.231Z&amp;apos;
System info: host: &amp;apos;DESKTOP-CL5VKH3&amp;apos;, ip: &amp;apos;192.168.1.15&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;11.0.10&amp;apos;
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: The Android SDK root folder &amp;apos;C:\Users\jeanl\AppData\Local\Android\Sdkxxx&amp;apos; does not exist on the local file system. Read https://developer.android.com/studio/command-line/variables for more details
    at getResponseForW3CError (C:\Users\jeanl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Users\jeanl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)
Build info: version: &amp;apos;3.13.0&amp;apos;, revision: &amp;apos;2f0d292&amp;apos;, time: &amp;apos;2018-06-25T15:24:21.231Z&amp;apos;
System info: host: &amp;apos;DESKTOP-CL5VKH3&amp;apos;, ip: &amp;apos;192.168.1.15&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;11.0.10&amp;apos;
Driver info: driver.version: AndroidDriver">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: The Android SDK root folder 'C:\Users\jeanl\AppData\Local\Android\Sdkxxx' does not exist on the local file system. Read https://developer.android.com/studio/command-line/variables for more detailsBuild info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:24:21.231Z'System info: host: 'DESKTOP-CL5VKH3', ip: '192.168.1.15', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.10'Driver info: driver.version: AndroidDriverremote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: The Android SDK root folder 'C:\Users\jeanl\AppData\Local\Android\Sdkxxx' does not exist on the local file system. Read https://developer.android.com/studio/command-line/variables for more details    at getResponseForW3CError (C:\Users\jeanl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)    at asyncHandler (C:\Users\jeanl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:24:21.231Z'System info: host: 'DESKTOP-CL5VKH3', ip: '192.168.1.15', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.10'Driver info: driver.version: AndroidDriverat io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:216)at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:225)at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:247)at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:212)at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:338)at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:130)at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:37)at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:89)at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:110)at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:134)at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:174)at jl.project.__commontests.TestsUtilCommon.setup(TestsUtilCommon.java:46)at jl.project.androidtests.UserRequirement1_Test.setup(UserRequirement1_Test.java:31)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.base/java.lang.reflect.Method.invoke(Method.java:566)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:163)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)Caused by: java.lang.reflect.InvocationTargetExceptionat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.base/java.lang.reflect.Method.invoke(Method.java:566)at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:193)... 42 moreCaused by: org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: The Android SDK root folder 'C:\Users\jeanl\AppData\Local\Android\Sdkxxx' does not exist on the local file system. Read https://developer.android.com/studio/command-line/variables for more detailsBuild info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:24:21.231Z'System info: host: 'DESKTOP-CL5VKH3', ip: '192.168.1.15', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.10'Driver info: driver.version: AndroidDriverremote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: The Android SDK root folder 'C:\Users\jeanl\AppData\Local\Android\Sdkxxx' does not exist on the local file system. Read https://developer.android.com/studio/command-line/variables for more details    at getResponseForW3CError (C:\Users\jeanl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)    at asyncHandler (C:\Users\jeanl\AppData\Roaming\npm\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$new$0(W3CHandshakeResponse.java:57)at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$getResponseFunction$2(W3CHandshakeResponse.java:104)at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:123)at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:127)at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:543)at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:126)... 47 more]]>
    </failure>
  </testcase> <!-- @BeforeClass setup -->
  <testcase classname="jl.project.androidtests.UserRequirement1_Test" name="@BeforeMethod navigate" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod navigate -->
  <testcase classname="jl.project.androidtests.UserRequirement1_Test" name="@AfterClass releaseResources" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterClass releaseResources -->
  <testcase classname="jl.project.androidtests.UserRequirement1_Test" name="createCategory" time="0.0">
    <skipped/>
  </testcase> <!-- createCategory -->
</testsuite> <!-- Default test -->
